### PR TITLE
iox-#743 Update top level README.md

### PR DESCRIPTION
**Pull request by elBoberido**
## Pre-Review Checklist for the PR Author

1. [x] Code follows the coding style of [CONTRIBUTING.md][contributing]
1. [x] Tests follow the [best practice for testing][testing]
1. [x] Changelog updated [in the unreleased section][changelog] including API breaking changes
1. [x] Branch follows the naming format (`iox-#123-this-is-a-branch`)
1. [x] Commits messages are according to this [guideline][commit-guidelines]
    - [x] Commit messages have the issue ID (`iox-#123 commit text`)
    - [x] Commit messages are signed (`git commit -s`)
    - [x] Commit author matches [Eclipse Contributor Agreement][eca] (and ECA is signed)
1. [x] Update the PR title
   - Follow the same conventions as for commit messages
   - Link to the relevant issue
1. [x] Relevant issues are linked
1. [x] Add sensible notes for the reviewer
1. [x] All checks have passed (except `task-list-completed`)
1. [x] Assign PR to reviewer

[commit-guidelines]: https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
[eca]: http://www.eclipse.org/legal/ECA.php
[contributing]: https://github.com/eclipse-iceoryx/iceoryx/blob/master/CONTRIBUTING.md#coding-style
[testing]: https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/website/advanced/best-practice-for-testing.md
[changelog]: https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/website/release-notes/iceoryx-unreleased.md

## Checklist for the PR Reviewer

- [x] Commits are properly organized and messages are according to the guideline
- [x] Code according to our coding style and naming conventions
- [x] Unit tests have been written for new behavior
    - [x] Each unit test case has a unique UUID
- [x] Public API changes are documented via doxygen
- [x] Copyright owner are updated in the changed files
- [x] PR title describes the changes

## Post-review Checklist for the PR Author

1. [x] All open points are addressed and tracked via issues

## References

- Relates to #743

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,12 +13,13 @@
 
 ## Introduction
 
-Great that you've made it to this little Eclipse project! Let's get you started by providing a quick background
+Great that you've made it to this neat Eclipse project! Let's get you started by providing a quick background
```
Comment (elfenpiff) [inline README.md:16]: Nice choice of words.

File: README.md [hunk 2]
```diff
@@ -13,12 +13,13 @@
...
 tour, introducing the project scope and all you need for installation and a first running example.
 
 So first off: What is iceoryx?
 
-iceoryx is an inter-process-communication (IPC) middleware for various operating systems (currently we support Linux, MacOS, QNX and Windows 10).
+iceoryx is an inter-process-communication (IPC) middleware for various operating systems (currently we support Linux,
+macOS, QNX, FreeBSD and Windows 10).
```
Comment (elfenpiff) [inline README.md:22]: FreeBSD uuuh yeah :D 

But maybe we can state that we should support unix posix systems in general but run our CI only on FreeBSD as a representative.

**Review by elfenpiff — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (elBoberido) [reply README.md:22]: That one is explicitly mentioned further down in the `Supported Platforms` section

File: README.md [hunk 3]
```diff
@@ -35,11 +36,10 @@ This ensures data transmissions with constant latency, regardless of the size of
 <img src="https://user-images.githubusercontent.com/55156294/91751530-35af7f80-ebc5-11ea-9aed-eed590b229df.png" width="80%">
 </p>
 
-You're right, middleware is a cluttered term and can somehow be all or nothing, so let's talk about the [goals and non-goals](doc/goals-non-goals.md) of iceoryx.
+You're right, middleware is a cluttered term and can somehow be all or nothing. To get a better impression what
```
[the remark this answers is not in the document]
Comment (elBoberido) [reply README.md:39]: done

**Review by FerdinandSpitzschnueffler — APPROVED** [review]

**Review by elfenpiff — APPROVED** [review]

**Pull request merged**